notebooks/depth_time_series.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
<p>Is it possible to pass these parameters to the API so that we only download records with these values?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Thank you for the detailed review! Actually, passing <code>startdepth</code>  = -1.0 would fetch all records with non-NaN depth values. Similarly, defining a suitable <code>startdate</code> parameter would help us fetch non-NaN values but I thought of passing no <code>startdate</code> because I couldn't figure out a suitable one that would work for every species. I relied on pandas to clear off NaN values. </p>